### ISSUE TITLE
[BUG] <title> No heartbeat received from MQLLMEngine

### ISSUE THREAD

**Issue by hulk-zhk** [QwenLM/Qwen#1322]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

PyTorch version: 2.4.0+cu121
 Python version: 3.10.12
启动模型服务：
python3 -m vllm.entrypoints.openai.api_server --host [0.0.0.0](http://0.0.0.0/) \
--port 10023 --seed 1024 \
--served-model qwen1.5-4b \
--model "/home/work/ssd1/Qwen1.5-4B-Chat" --trust-remote-code \
--tokenizer-mode slow --max-model-len 128 \
--dtype float16 
进行模型推理：
curl http://127.0.0.1:10023/v1/completions \
-H "Content-Type: application/json" \
-d '{
"model":"qwen1.5-4b", "max_tokens":64, "seed":0, "top_k":1, "top_p":0.8, "temperature":0.3, 
"prompt": "谈谈你对大模型的理解"
}'
报错信息：
No heartbeat received from MQLLMEngine

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```
